ibm_db2.c

```diff
@@ -960,8 +960,8 @@ PHP_MINIT_FUNCTION(ibm_db2)
     REGISTER_LONG_CONSTANT("DB2_PARAM_FILE", 11, CONST_CS | CONST_PERSISTENT);
     REGISTER_LONG_CONSTANT("DB2_TRUSTED_CONTEXT_ENABLE", SQL_ATTR_USE_TRUSTED_CONTEXT, CONST_CS | CONST_PERSISTENT);
 
-    REGISTER_LONG_CONSTANT("DB2_AUTOCOMMIT_ON", SQL_AUTOCOMMIT_ON, CONST_CS | CONST_PERSISTENT);
-    REGISTER_LONG_CONSTANT("DB2_AUTOCOMMIT_OFF", SQL_AUTOCOMMIT_OFF, CONST_CS | CONST_PERSISTENT);
+    REGISTER_BOOL_CONSTANT("DB2_AUTOCOMMIT_ON", SQL_AUTOCOMMIT_ON, CONST_CS | CONST_PERSISTENT);
+    REGISTER_BOOL_CONSTANT("DB2_AUTOCOMMIT_OFF", SQL_AUTOCOMMIT_OFF, CONST_CS | CONST_PERSISTENT);
```

Comment:
Would it make sense to keep them integer and change the signature of the `db2_autocommit()`? Even if not currently exposed, there are other modes defined in `include/sqlext.h`:
```c
/* SQL_AUTOCOMMIT options */
#define SQL_AUTOCOMMIT_OFF              0UL
#define SQL_AUTOCOMMIT_ON               1UL
#define SQL_AUTOCOMMIT_DEFERRED         2UL
```

Reply:
Changing the signature was my first plan of attack, but since it currently does no validation on the value passed in, I figured it would be fewer changes this way. In light of `SQL_AUTOCOMMIT_DEFERRED`, I think it does make sense to leave it open for future enhancements and change the signature.